makefile.shade

```diff
@@ -60,6 +60,7 @@ var MANAGED_PROJECTS = '${FindAllProjects(
     "Microsoft.Framework.PackageManager",
     "Microsoft.Framework.Project",
     "Microsoft.Framework.Runtime",
+	"Microsoft.Framework.Runtime.Caching",
```

Comment:
I think we still have the issue with `dnu list --assemblies` not picking up project outputs...